src/main/native/pairhmm/IntelPairHmm.cc

```diff
@@ -128,9 +144,6 @@ JNIEXPORT void JNICALL Java_com_intel_gkl_pairhmm_IntelPairHmm_computeLikelihood
     DBG("result = %e", result_final);
   }
 
-  //==================================================================
-  // release Java data
-  javaData.releaseData(env);
```

Comment:
This should stay to release data/memory

Reply:
I deliberately moved it into the destructor so it's not possible to forget about running this. More importantly, this also means there's no need to call this explicitly when an exception gets triggered (it happens automatically during stack unwinding).

Reply:
I see, may be add summary about changes to provide context.